access-modifier-checker/src/it/failOnError/caller/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>test</groupId>
+        <artifactId>failOnError</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>caller</artifactId>
+    <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+    </dependencies>
+    <properties>
+        <access-modifier-checker.failOnError>false</access-modifier-checker.failOnError>
```

Comment:
Wanted one IT using the mojo configuration, and one using the Maven property, to make sure I got the syntax right in the magic Javadoc comment. Would be better to update to the Maven mojo annotations library at some point, but I did not want to confuse things with that bigger change.